### ISSUE TITLE
[WCAG] Ensure keyboard navigation - Details Panel

### ISSUE THREAD

**Issue by kenchase** [Canadian-Geospatial-Platform/geoview#3217]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Current Behavior

Review tab order of all focusable elements within the panel

1 - When pressing the info/guide close button, focus is not returned to the corresponding layer list item (see attached screenshot)

2 - The info/guide close button should be hidden or disabled when no features have been selected.

3 - When multiple features are available, pressing a layer should autofocus on the "next" icon button.

4 - Prev/Next Icon Button: Focus is not obvious when last item is pressed

<img width="517" height="491" alt="Image" src="https://github.com/user-attachments/assets/5839b608-4dd1-4464-84d0-d1f6c71e763a" />

### Expected Behavior

When pressing the info/guide close button, focus should be returned to the corresponding layer list item.
